### PR TITLE
generate_mask: mask velocity based on velocityStd

### PR DESCRIPTION
**Pull request by forrestfwilliams**
References issue #613
Generate mask based on formula |velocity|>2*velocitStd

**Description of proposed changes**
- adds --vstd argument
- adds mask generation functionality in create_threshold_mask()

**Reminders**
- [x] Pass Codacy code review (green)
- [x] Pass Circle CI test (green)
- [x] Make sure that your code follows our style. Use the other functions/files as a basis.
- [x] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

File: mintpy/generate_mask.py [hunk 1]
```diff
@@ -69,6 +72,8 @@ def create_parser():
                         help='maximum value for selected pixels')
     parser.add_argument('-p','--mp','--minpixels', dest='minpixels', type=int,
                         help='minimum cluster size in pixels, to remove small pixel clusters.')
+    parser.add_argument('--vstd', action='store_true',
+                        help='mask according to the formula: |velocity|>2*velocityStd')
```
Comment (yunjunz) [inline mintpy/generate_mask.py:76]: Could you add an extra option to specify the cutoff number, e.g. `--vstd-num`, with a default value of 2?

Comment (forrestfwilliams) [reply mintpy/generate_mask.py:76]: Sure!

**Review by yunjunz — APPROVED** [review]
Looks great, thank you @forrestfwilliams!

**Comment by forrestfwilliams** [comment]
No problem @yunjunz! Thanks for being willing to include this niche functionality within MintPy!

**Pull request merged**